registry/proxy/proxyblobstore.go

```diff
@@ -67,6 +68,26 @@ func (pbs *proxyBlobStore) copyContent(ctx context.Context, dgst digest.Digest,
 	return desc, nil
 }
 
+// doServeBlobResponse is intended to avoid actual datacopy of Head request.
+// As it serves both Get and Head http request.
+func (pbs *proxyBlobStore) doServeBlobResponse(ctx context.Context, dgst digest.Digest, writer http.ResponseWriter, reqMethod string) error {
+	switch reqMethod {
+	case http.MethodHead:
+		desc, err := pbs.remoteStore.Stat(ctx, dgst)
+		if err != nil {
+			return err
+		}
+		setResponseHeaders(writer, desc.Size, desc.MediaType, dgst)
+	case http.MethodGet:
+		if _, err := pbs.copyContent(ctx, dgst, writer); err != nil {
+			return err
+		}
+	default:
+		return errors.Errorf("blobstore: unsupported request method[%s]\n", reqMethod)
```

Comment:
could you use `fmt.Errorf` instead? That should fix the compile errors in CircleCI.

Also looking at existing error messages, it's not prevalent to use `fmt.Errorf("blobstore: unsupported request method: %s", reqMethod)`